js/huobi.js

```diff
@@ -4752,41 +4766,17 @@ module.exports = class huobi extends Exchange {
         //     }
         //
         const data = this.safeValue (response, 'data', []);
-        const parsed = this.parseDepositAddresses (data, [ code ], false);
-        return this.indexBy (parsed, 'network');
```

Comment:
as described in second comment, this was conflicting, as we should return only one